sdk/communication/azure-communication-chat/src/main/java/com/azure/android/communication/chat/signaling/TrouterUtils.java

```diff
@@ -389,38 +420,12 @@ private static String parseShareHistoryTime(Long shareHistoryTimeEpochMilli) {
             .toString();
     }
 
-    private static String parseRecipientSkypeId(String skypeId) {
-        if (skypeId.startsWith(USER_PREFIX)) {
-            return skypeId;
+    private static String parseRecipientRawId(String rawId) {
```

Comment:
I think I misunderstood what this function does originally. It seems to be the temporary workaround for when the prefix is missing in the id. If that is the case, you can call it `addRawIdPrefixIfMissing` which would explains what it does literarily.

Reply:
And once we update the template, we can remove this